### PR TITLE
[ADF - 1030] Datatable - enable isLoading to show the spinner in all the components

### PR DESCRIPTION
**Pull request by infad-kachancheri**
- added loader in tasklist and task-attachment-list component
- added loader in processlist, process-variable-list and process-attachment-list component

### CODE REVIEW

**Comment by infad-kachancheri** [comment]
rebase done

File: ng2-components/ng2-activiti-tasklist/src/components/adf-task-attachment-list.component.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export class TaskAttachmentListComponent implements OnChanges {
     error: EventEmitter<any> = new EventEmitter<any>();
 
     attachments: any[] = [];
+    loadingFlag: boolean = true;
```
Comment (eromano) [inline ng2-components/ng2-activiti-tasklist/src/components/adf-task-attachment-list.component.ts]: loadingFlag can we call it isLoading?

File: ng2-components/ng2-activiti-processlist/src/components/activiti-process-instance-variables.component.ts [hunk 2]
```diff
@@ -61,6 +61,8 @@ export class ActivitiProcessInstanceVariables implements OnInit, OnChanges {
     variableValue: string;
     variableScope: string;
 
+    loadingFlag: boolean = true;
```
Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-process-instance-variables.component.ts]: loadingFlag can we call it isLoading ?

File: ng2-components/ng2-activiti-processlist/src/components/activiti-processlist.component.ts [hunk 3]
```diff
@@ -62,6 +62,7 @@ export class ActivitiProcessInstanceListComponent implements OnChanges, AfterCon
     onError: EventEmitter<any> = new EventEmitter<any>();
 
     currentInstanceId: string;
+    loadingFlag: boolean = true;
```
Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-processlist.component.ts]: loadingFlag can we call it isLoading?

Comment (infad-kachancheri) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-processlist.component.ts]: already method name is 'isLoading'

Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-processlist.component.ts]: you don't need a method isLoading that return loadingFlag. Or you create a get that return isLoading or you just use the property

**Comment by mauriziovitale** [comment]
@eromano Can you merge it, please? I'll change the property name in a different issue

**Pull request merged**